### PR TITLE
Feature/add trees to treecluster in watering plan

### LINKED ISSUE #283: Remove not used entity types

### PR DESCRIPTION
**Pull request by doriengr**
Close #283
refs #260

### CODE REVIEW

File: internal/storage/postgres/flowerbed/flowerbed.go [hunk 1]
```diff
@@ -38,7 +38,6 @@ func NewFlowerbedMappers(
 }
 
 func NewFlowerbedRepository(s *store.Store, mappers FlowerbedMappers) storage.FlowerbedRepository {
-	s.SetEntityType(store.Flowerbed)
```
Comment (choffmann) [inline internal/storage/postgres/flowerbed/flowerbed.go:41]: Yes, away with it :raised_hands:

File: internal/storage/postgres/store/store.go [hunk 2]
```diff
@@ -11,23 +11,9 @@ import (
...
-const (
-	Sensor       EntityType = "sensor"
-	Image        EntityType = "image"
-	Flowerbed    EntityType = "flowerbed"
-	TreeCluster  EntityType = "treecluster"
-	Tree         EntityType = "tree"
-	Vehicle      EntityType = "vehicle"
-	Region       EntityType = "region"
-	WateringPlan EntityType = "watering plan"
-)
```
Comment (choffmann) [inline internal/storage/postgres/store/store.go:25]: Love it. Finally someone who refactors :heart_hands:

**Review by choffmann — APPROVED** [review]
LGTM :+1:

File: internal/storage/postgres/flowerbed/flowerbed.go (hunk 1 above)
Comment (doriengr) [reply internal/storage/postgres/flowerbed/flowerbed.go:41]: Wuhuuuuuu xD

**Pull request merged**